test/unittests/test_analyse_xml_data/test_construct_and_analyze_graph.py

```diff
@@ -1,15 +1,26 @@
-# pylint: disable=missing-class-docstring,missing-function-docstring,protected-access
+# pylint: disable=missing-class-docstring,missing-function-docstring,protected-access,no-member
+# mypy: disable-error-code="var-annotated,assignment,arg-type"
```

Comment:
how many of those can you get rid of, if you don't use `patch`?

Reply:
None, the last pylint is everytime we use rx instances. The other ones I don't remember exactly but they are not about the patch.